### ISSUE TITLE
Need regeneration of rdoc-data for Ruby 2.1

### ISSUE THREAD

**Issue by headius** [rdoc/rdoc#303]
Error running rdoc-data says it all:

```
Your ruby version 2.1.2 is not supported, only 1.8.7, 1.9.2, 1.9.3, 2.0.0
```

**Comment by zzak** [comment]
Ugh, it has been over a year..

**Comment by headius** [comment]
JRuby does not ship rdoc data in any form, so we are very eagerly awaiting an update too :-)

**Comment by zzak** [comment]
Eric just gave me access to rdoc-data so I'll see what I can do about updating the gem.

Thanks for your patience everyone!

**Comment by zzak** [comment]
This should be fixed by using rdoc-data 4.1.0.

Thanks for your patience! <3
